ivy/data_classes/array/experimental/losses.py

```diff
@@ -258,7 +258,7 @@ def kl_div(
         /,
         *,
         reduction: Optional[str] = "mean",
-        out: Optional[ivy.Array] = None,
+        log_target=False,
```

Comment:
I don't know about this change 🤔 why did you remove the `out` parameter and introduce a new parameter in the functional API of ivy?

Reply:
You can not remove `out` parameter. It's part of the standard for API that we are following all across the Functional API

Reply:
I'll add back the `out` parameter, but as for the `log_target`, aren't we trying to capture the superset behaviour? If so then I think it is important to include it, given that `torch` implements it